include/libmesh/Makefile.am

```diff
@@ -204,6 +205,54 @@ BUILT_SOURCES = \
         unv_io.h \
         vtk_io.h \
         xdr_io.h \
+        compare_types.h \
```

Comment:
And how did this happen?  You don't actually have these files in include/ anywhere, right?  They're all in contrib?